modules/index.js

```diff
@@ -35,9 +35,12 @@ const blocksMO = window.MutationObserver ? new MutationObserver(blocksMCB)
 const mediaMO = window.MutationObserver ? new MutationObserver(mediaMCB)
   /* c8 ignore next */ : {};
 
+let maxEvents = 1023;
+
 function trackCheckpoint(checkpoint, data, t) {
   const { weight, id } = window.hlx.rum;
-  if (isSelected) {
+  if (isSelected && maxEvents > 0) {
```

Comment:
```suggestion
  if (isSelected && --maxEvents > 0) {
```

If ESLint doesn't complain, then I'd one-liner this.

Reply:
```suggestion
  if (isSelected && --maxEvents) {
```
every byte counts.

Reply:
yes, lint is complaining for `no-plusplus`, so, either we need to add the eslint-disable or avoid `--`, I'll be using the latter.